updates/22.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE einzelbeleg DROP FOREIGN KEY einzelbeleg_QuelleGattungID;
+-- this is necessary because the id column contains 0 values. Otherwise an error appears
+ALTER TABLE einzelbeleg ADD CONSTRAINT einzelbeleg_QuelleGattungID FOREIGN KEY (QuelleGattungID) REFERENCES selektion_quellengattung(ID) ON UPDATE CASCADE;
```

Comment:
In the meantime, another Foreign key has been added for hierarchies which needs to be considered in this update as well => selektion_quellengattung.parentId is referencing selektion_quellengattung.ID

Reply:
I have added the foreign key to the script